locales/el.yml

```
@@ -52,6 +52,7 @@ el:
       libraries: 'Libraries'
       examples: 'Examples'
       faq: 'FAQ'
+      new_analytics_tracking_script_migration_guide: 'New Analytics Tracking Script Migration Guide'
```

And change the key to `new_script_migration_guide`

```suggestion
      new_script_migration_guide: 'New Analytics Tracking Script Migration Guide'
```